tests/globals.js

```diff
@@ -414,6 +415,5 @@ global.jet = {
   },
 };
 
-// TODO
-global.isCI = false;
-// global.isCI = !!process.env.CI;
+// TODO toggle this correct in CI only.
+global.isCI = true;
```

Comment:
@Salakar is this running in app context only now? That is, in Simulator/Emulator/macOS-app ? If so, we could pass env vars to the app in DYLD style for the Apple ones and I'm sure there's a similar way on Android I think so that we can go conditional on them - not sure if there is another viable communication mechanism to transmit through from "yarn run package.json run script" --> in app visible thing

Reply:
We can pass envs through Jet context feature now just not had time to set it up